cime_config/machines/cmake_macros/oneapi-ifxgpu.cmake

```diff
@@ -4,15 +4,17 @@ if (compile_threaded)
   string(APPEND CMAKE_CXX_FLAGS " -qopenmp")
   string(APPEND CMAKE_EXE_LINKER_FLAGS  " -qopenmp")
 endif()
-string(APPEND CMAKE_C_FLAGS_RELEASE   " -O2")
-string(APPEND CMAKE_Fortran_FLAGS_RELEASE   " -O2")
-string(APPEND CMAKE_CXX_FLAGS_RELEASE " -O2")
-string(APPEND CMAKE_Fortran_FLAGS_DEBUG   " -O0 -g -fpe0")
+
+#adding -g here leads to linker internal errors
```

Comment:
And yet I see `-g` below. Is this comment outdated, and link is fine now? If so, we should remove it to avoid confusion.

Reply:
"just -g" is in debug flags only, and i haven't tried to build debug. so i only changed release flags.

i was able to compile and to run with "just -g" on sunspot, however, the folder size for e3sm build gets from say 4 gb to 140 gb with full -g. last i tried "just -g" on aurora, there were linking issues. so this is all WIP. i'd say keep this as is now and sort it later.

Reply:
What I meant is that the comment is confusing: it makes it sound like we should not use `-g` b/c of link errors, but we _do_ use it across the board. Maybe the comment needs to be clarified. Something like "WARNING: the -g flag seems to have a wild behavior sometimes, with side effects ranging from VERY large binaries to link errors. If you experience errors, remove it".